src/schema/v2/show.ts

```diff
@@ -594,6 +594,10 @@ export const ShowType = new GraphQLObjectType<any, ResolverContext>({
         resolve: (show, args, context) =>
           followArtistsResolver({ show_id: show.id }, args, context),
       },
+      viewingRoomIDs: {
+        type: new GraphQLList(GraphQLString),
```

Comment:
Can you make these both non-nullable?

Reply:
(Like this should return empty array instead of null if there is none)

Reply:
yup, updating.